lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -219,6 +227,23 @@ def get_redcap_api_token(api_url: str) -> str:
     return token
 
 
+def get_redcap_records(project: Project, record_ids: List[str], raw: bool) -> Optional[List[Record]]:
```

Comment:
I think, if given a an empty list of `record_ids`, we would want this function to return without fetching from REDCap. The current behavior is that I get a log output of 
```
[2020-05-12 13:52:00] Fetching REDCap project 20759
[2020-05-12 13:52:01] Fetching 0 REDCap records from project 20759
```
followed by a long pause, then a bunch of log statements about duplicate REDCap record IDs. These aren't real duplicate REDCap record IDs but the result of the `KeyError` exception where we're processing records returned from REDCap but don't have those records tracked locally in `latest_complete_dets`.

Reply:
This makes me wonder if `get_redcap_records` is doing what we want; if we're passing it an empty list of `record_ids`, why are so many `records` being returned? (e.g. 16619 for `scan-en`)

Reply:
OH. If you don't provide `record_id` to the REDCap API then it returns all records by default.

Reply:
Updated in latest commit